docs/linux/setup_ubuntu-host_qemu-vm_x86-64-kernel.md

```diff
@@ -36,10 +36,10 @@ ls $GCC/bin/
 
 Command:
 ``` bash
-git clone --branch v5.14 git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git $KERNEL
+git clone --branch v6.2.2 git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git $KERNEL
```

Comment:
This tag does not exist on torvalds. It's a stable tag. torvalds/linux.git only has vMajor.Minor along with vMajor.Minor-rcX tags.

Let's just suggest v6.2.